Description: Enable reproducible builds
Author: Stephen Kitt <skitt@debian.org>

Allow reproducible builds by disabling timestamps in the generated
documentation.

--- a/doc/libevdev.doxygen.in
+++ b/doc/libevdev.doxygen.in
@@ -1129,7 +1129,7 @@
 # The default value is: NO.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_TIMESTAMP         = YES
+HTML_TIMESTAMP         = NO
 
 # If the HTML_DYNAMIC_SECTIONS tag is set to YES then the generated HTML
 # documentation will contain sections that can be hidden and shown after the
